### PR TITLE
main_readme push - master

### PR DESCRIPTION
**Pull request by X3NOSIZ**

### CODE REVIEW

**Review by Neilblaze — COMMENTED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -0,0 +1,2229 @@
...
+      * [Ruby scripts](#ruby-scripts)
+   * [Books and other resources](#books-and-other-resources)
+   * [Bug Reports and Feature Requests](#bug-reports-and-feature-requests)
+   * [Contribution Guidelines](#contribution-guidelines)
+   * [Author](#author)
+   * [License](#license)
+
+
+
+Installation
```
Comment (Neilblaze) [inline README.md:128]: Could have been added in later section

File: README.md [hunk 2]
```diff
@@ -0,0 +1,2229 @@
...
+```ruby
+# single line comment
+
+=begin
+multiline
+comment
+=end
+```
+
+Operators
```
Comment (Neilblaze) [inline README.md:258]: Later x Cross #2

File: README.md [hunk 3]
```diff
@@ -0,0 +1,2229 @@
...
+
+Bug Reports and Feature Requests
+============
+Please create an issue with as much information you can. Thank you.
+
+Contribution Guidelines
+============
+<a href="https://github.com/lifeparticle/Ruby-Cheatsheet/blob/master/CONTRIBUTING.md"><img alt="contribution guidelines" src="https://img.shields.io/badge/contribution-guidelines-brightgreen.svg?style=flat"/></a>
+
+Author
```
Comment (Neilblaze) [inline README.md:2223]: Could have included Github link

**Review by Neilblaze — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (X3NOSIZ) [reply README.md:2223]: Ok! on it!

**Review by X3NOSIZ — COMMENTED** [review]
Objectives OK!

File: README.md (hunk 2 above)
Comment (X3NOSIZ) [reply README.md:258]: Added!

**Review by Neilblaze — APPROVED** [review]

**Pull request merged**